### PR TITLE
 Added code of Coding Exercise-23

### PR DESCRIPTION
**Pull request by anushreeSonak**

### CODE REVIEW

File: First/dailyExercises/AnagramREADme.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Write a Java program that checks whether two strings are anagrams of each other:
+
+OUTPUT:
+"C:\Program Files\Java\jdk-17\bin\java.exe" "-javaagent:C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2023.1\lib\idea_rt.jar=61144:C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2023.1\bin" -Dfile.encoding=UTF-8 -classpath "C:\Users\anushrees\Desktop\New folder\DemoRepo\First\out\production\First1" firstPackage.First.dailyExercises.AnagramChecker
+listen and silent are anagrams.
```
Comment (waghmaresuyog) [inline First/dailyExercises/AnagramREADme.md:5]: Remove this line

File: First/dailyExercises/AnagramREADme.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Write a Java program that checks whether two strings are anagrams of each other:
+
+OUTPUT:
+"C:\Program Files\Java\jdk-17\bin\java.exe" "-javaagent:C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2023.1\lib\idea_rt.jar=61144:C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2023.1\bin" -Dfile.encoding=UTF-8 -classpath "C:\Users\anushrees\Desktop\New folder\DemoRepo\First\out\production\First1" firstPackage.First.dailyExercises.AnagramChecker
```
Comment (waghmaresuyog) [inline First/dailyExercises/AnagramREADme.md]: Remove this unwanted line

Comment (anushreeSonak) [reply First/dailyExercises/AnagramREADme.md]: DONE !

**Review by waghmaresuyog — APPROVED** [review]
Approving PR

**Review by prashantkawa — APPROVED** [review]

**Pull request merged**